### PR TITLE
Genecut

### PR DESCRIPTION
**Pull request by DmitriiSukhomlinov**
GeneCut desktop plugin. 
This plugin is needed to access GeneCut calculation results and easily show them into UGENE. Features available:
1. Use the opened sequence in GeneCut
2. Check all calculation results
3. Brief description on the selected calculation
4. Open the full GeneCut report on the selected calculation
5. Get the input sequence of on the selected calculation
6. Get the result sequence on the selected calculation (optimized sequence with annotations - oligos and long blocks)
7. Compare the input and the output sequences of calculation

Results are tied with your user, so you need to login before work.
GeneCut server: http://genecut.unipro.ru/ create a user to test (also possible to create user from UGENE). There is a detailed manual (for now Russian only, English is expected). In case of questions ask me.

12 GUI test are added, no XML test because there are no functional which could be tested with this type of tests. Basically, this plugin is just an HTML-request-sender and their-responds-parser.

### CODE REVIEW

**Comment by DmitriiSukhomlinov** [comment]
TODO: update bundles for other systems

**Comment by yalgaer** [comment]
I need ~1 day to build and test this PR. Stay tuned!

File: src/plugins/GUITestBase/GUITestBase.pro [hunk 1]
```diff
@@ -496,6 +499,7 @@ SOURCES +=  src/GUITestBasePlugin.cpp \
             src/tests/common_scenarios/create_shortcut/GTTestsCreateShortcut.cpp \
             src/tests/common_scenarios/document_from_text/GTTestsDocumentFromText.cpp \
             src/tests/common_scenarios/dp_view/GTTestsDpView.cpp \
+			src/tests/common_scenarios/genecut/GTTestsGeneCut.cpp \
```
Comment (yalgaer) [inline src/plugins/GUITestBase/GUITestBase.pro:502]: nit: fix formatting

**Comment by yalgaer** [comment]
Please run GUI tests on Linux and I will approve

**Comment by DmitriiSukhomlinov** [comment]
@yalgaer done. I need to merge it into release or master?

**Review by yalgaer — APPROVED** [review]

**Pull request merged**